crates/staged-sync/src/utils/versioning/db.rs

```diff
@@ -0,0 +1,100 @@
+use crate::utils::versioning::client::CLIENT_VERSION;
+use std::{
+    fs, io,
+    path::{Path, PathBuf},
+};
+
+/// The name of the file that contains the version of the database.
+const DB_VERSION_FILE_NAME: &str = "database.version";
```

Comment:
This is database stuff so it should be in the database crate, not in here imo (especially since we want to axe this crate)

Reply:
moved to `storage` crate, but left the `init_db` helper function in `staged-sync` crate, let's fix it in a separate PR